daemon/images/image_commit.go

```diff
@@ -109,7 +110,7 @@ func exportContainerRw(layerStore layer.Store, id, mountLabel string) (arch io.R
 //   - it doesn't log a container commit event
 //
 // This is a temporary shim. Should be removed when builder stops using commit.
-func (i *ImageService) CommitBuildStep(c backend.CommitConfig) (image.ID, error) {
+func (i *ImageService) CommitBuildStep(ctx context.Context, c backend.CommitConfig) (image.ID, error) {
 	ctr := i.containers.Get(c.ContainerID)
```

Comment:
For others reviewing; looks like there's probably no need to pass the context further down here (or at least, not in the current implementation); https://github.com/moby/moby/blob/219ced474d8bcb9cbd399a6ab6ec17f178fe3306/container/memory_store.go#L29-L35

Reply:
This PR is basically an intro to the c8d commit, which will use this one. I decided to post this first because the commit itself still needs a bit more love.